x-pack/filebeat/input/entityanalytics/provider/jamf/jamf.go

```diff
@@ -374,7 +375,9 @@ func (p *jamfInput) runIncrementalUpdate(inputCtx v2.Context, store *kvstore.Sto
 	if len(updatedDevices) != 0 {
 		tracker = kvstore.NewTxTracker(ctx)
 		for _, d := range updatedDevices {
-			p.publishComputer(d, inputCtx.ID, client, tracker)
+			if d.Modified.After(last) {
```

Comment:
It looks like this check is redundant, because we already know that `updatedDevices` only has the ones that are absent or different in the store.

Given that, I'm not sure anything needs to be done for Jamf.

Reply:
Agreed. I'm going to back out the Jamf changes. We can come back to this later.